modin/pandas/test/test_io.py

```diff
@@ -173,48 +189,191 @@ def teardown_test_file(test_path):
         os.remove(test_path)
 
 
-@pytest.fixture
-def make_csv_file(delimiter=",", compression="infer"):
-    """Pytest fixture factory that makes temp csv files for testing.
+def get_random_string():
+    random_string = "".join(
+        random_state.choice(
+            [x for x in ascii_uppercase + ascii_lowercase + digits], size=10
```

Comment:
Why is it so hard to create unique strings?

I mean, can't we get by with a smaller set of characters from which strings will be created?

Reply:
I'm not sure if I understood you. We can create test string from smaller set of characters, but why we should do?

Reply:
This is Occam's blade. Doing what can be easier is easier.

For example: use only string.ascii_letters.

Reply:
Ok, only `string.ascii_letters` is used now.